common/scala/src/main/scala/whisk/core/entity/ActivationResult.scala

```diff
@@ -77,11 +74,11 @@ protected[core] object ActivationResponse extends DefaultJsonProtocol {
     protected[core] def success(result: Option[JsValue] = None) = ActivationResponse(Success, result)
 
     protected[core] def applicationError(errorValue: JsValue) = error(ApplicationError, errorValue)
-    protected[core] def applicationError(errorMsg: String)    = error(ApplicationError, JsString(errorMsg))
-    protected[core] def containerError(errorValue: JsValue)   = error(ContainerError, errorValue)
-    protected[core] def containerError(errorMsg: String)      = error(ContainerError, JsString(errorMsg))
-    protected[core] def whiskError(errorValue: JsValue)       = error(WhiskError, errorValue)
-    protected[core] def whiskError(errorMsg: String)          = error(WhiskError, JsString(errorMsg))
+    protected[core] def applicationError(errorMsg: String) = error(ApplicationError, JsString(errorMsg))
+    protected[core] def containerError(errorValue: JsValue) = error(ContainerError, errorValue)
+    protected[core] def containerError(errorMsg: String) = error(ContainerError, JsString(errorMsg))
+    protected[core] def whiskError(errorValue: JsValue) = error(WhiskError, errorValue)
+    protected[core] def whiskError(errorMsg: String) = error(WhiskError, JsString(errorMsg))
```

Comment:
there is an setting that gets the editor to align some tokens (e.g., `=>`), maybe there's another for `=` --- worth checking.